### DISCUSSION TITLE
Create a topic with an AutoRenewAccount without requiring an Admin Key.

### DISCUSSION THREAD

**Discussion by AdrianKBL** [hiero-ledger/hiero-improvement-proposals#984] [General]
Good morning,

Currently, it is not possible to create a topic with an **AutoRenewAccount** unless an **Admin Key** is also set. This requirement seems unnecessary and restrictive. I propose changing this policy to allow the setup of an **AutoRenewAccount** even if an Admin Key has not been configured.

Additionally, it should be possible to update an existing **AutoRenewAccount** to a new one. This update should require authorization from both the old and the new accounts signs in the transaction.

Thank you for considering these improvements.

**Comment by mgh14** [comment]
> This requirement seems unnecessary and restrictive.

@AdrianKBL can you provide more details, and perhaps a use case? It sounds dangerous to allow modification of a topic's **AutoRenewAccount** if there's no admin key set for that topic.

**Reply by AdrianKBL** [reply]
Hello @mgh14 

We are currently facing a significant issue where the absence of an Admin Key prevents the setup of an AutoRenew Account ID, which is illogical. In the case of Tokens, you can set up an AutoRenew account even without an Admin Key. When AutoRenew for topics is activated and payment is required to keep topics alive, what will happen to those topics without an Admin Key or an AutoRenew account?

If we have the signature of both the current account and the new account, that should suffice to update the AutoRenewAccount and resolve this problem.
